source/crud/crud.rst

```diff
@@ -1953,54 +1953,37 @@ in the collection prior to ``$merge`` being executed).
 Read preferences and server selection
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+This section is only applicable if an explicit (i.e. per-operation) or inherited
+(e.g. from a Collection) read preference is available and it is *not* a primary
+read preference (i.e. ``{ "mode": "primary" }``).
```

Comment:
Adding this paragraph up top allows us to briefly address the concept of an explicit or inherited read preference. From this point on, we don't actually care whether the RP is explicit/inherited -- there's either a RP available for this operation or there isn't (i.e. default to primary).

I've also noted that explicit RPs aren't discussed elsewhere in the spec apart from being alluded to in "Q: Where is read preference?", so removing repetitive mentions of that concept from this section seems like an improvement.